Services/WalletsService/Sources/WalletsService.swift

```diff
@@ -79,6 +80,10 @@ public struct WalletsService: Sendable {
         self.addressStatusService = addressStatusService
     }
 
+    public var walletsCount: Int {
```

Comment:
```suggestion
    public var walletsCount: Int {
```

I would recommend to add throwable method here to get the number and view model to handle try?